Cannot set title on the main page in Android

```
05-06 20:18:14.995 12707 12707 I chromium: [INFO:CONSOLE(310)] "Error in Error callbackId: NativeHost1158829928 : Error: Native error: java.lang.RuntimeException: Error setting FrameLayout's UserControl.Content to Windows.UI.Xaml.Controls.Page{d535279 V.E...... ......I. 0,0-0,0}
05-06 20:18:14.995 12707 12707 I chromium:      at run.ace.IncomingMessages.set(IncomingMessages.java:206)
05-06 20:18:14.995 12707 12707 I chromium:      at run.ace.NativeHost$2.run(NativeHost.java:239)
05-06 20:18:14.995 12707 12707 I chromium:      at android.os.Handler.handleCallback(Handler.java:739)
05-06 20:18:14.995 12707 12707 I chromium:      at android.os.Handler.dispatchMessage(Handler.java:95)
05-06 20:18:14.995 12707 12707 I chromium:      at android.os.Looper.loop(Looper.java:148)
05-06 20:18:14.995 12707 12707 I chromium:      at android.app.ActivityThread.main(ActivityThread.java:5422)
05-06 20:18:14.995 12707 12707 I chromium:      at java.lang.reflect.Method.invoke(Native Method)
05-06 20:18:14.995 12707 12707 I chromium:      at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:726)
05-06 20:18:14.995 12707 12707 I chromium:      at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:616)
05-06 20:18:14.995 12707 12707 I chromium: Caused by: java.lang.RuntimeException: Cannot set title on the main page in Android unless you set <preference name="ShowTitle" value="true"/> in config.xml.
05-06 20:18:14.995 12707 12707 I chromium:      at run.ace.TabBar.setTitle(TabBar.java:39)
05-06 20:18:14.995 12707 12707 I chromium:      at Windows.UI.Xaml.Controls.Page.updateTitle(Page.java:67)
05-06 20:18:14.995 12707 12707 I chromium:      at Windows.UI.Xaml.Controls.Page.processBars(Page.java:75)
05-06 20:18:14.995 12707 12707 I chromium:      at Windows.UI.Xaml.Controls.ViewGroupHelper.setProperty(ViewGroupHelper.java:51)
05-06 20:18:14.995 12707 12707 I chromium:      at run.ace.IncomingMessages.set(IncomingMessages.java:191)
05-06 20:18:14.995 12707 12707 I chromium:      ... 8 more
```

Already found [this thread on StackOverflow](http://stackoverflow.com/questions/35689252/use-ace-plugin-to-make-ios-android-tabbar-tableview-using-platform-specific-ja) but that does not help me finding the solution. I tried uppercase/lowercase `ShowTitle` in the config.xml, both in android specific config and global config. It just won't work. Do you have any idea?

--- a/src/plugin_hooks/android_before_compile.js
+++ b/src/plugin_hooks/android_before_compile.js
@@ -33,6 +33,12 @@ function copyAndroidFiles() {
 
 module.exports = function (context) {
     var fs = require('fs');
+    
+    // Do not allow theme with no action bar
+    var dest = path.join(__dirname, '../../../../platforms/android/AndroidManifest.xml');
+    var manifest = fs.readFileSync(dest, 'utf8');
+    manifest = manifest.replace('android:theme="@android:style/Theme.DeviceDefault.NoActionBar"', '');
+    fs.writeFileSync(dest, manifest);
 
     // Make sure the dependencies are installed
     try {

